### PR TITLE
Update Difficulties

### PR DESCRIPTION
**Pull request by circleen**

### CODE REVIEW

**Comment by circleen** [comment]
ready for review

File: wiki/Difficulties/en.md [hunk 1]
```diff
@@ -11,13 +11,14 @@ tags:
 
 *Not to be confused with [Beatmaps](/wiki/Beatmaps).*
 
-A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult it is.
+A beatmap's **difficulty** describes the level of skill needed to complete the beatmap. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult the beatmap is.
```
Comment (bdach) [inline wiki/Difficulties/en.md]: This reads worse to me after the change due to the stylistic repetition:

> a *beatmap's* difficulty (...) to complete the *beatmap*

> star rating of a *beatmap* (...) how difficult the *beatmap* is

The previous version using pronouns avoided this and was unambiguous in my opinion.

File: wiki/Difficulties/en.md [hunk 2]
```diff
@@ -11,13 +11,14 @@ tags:
...
+![The star rating on this map](img/difficulty-example.jpg)
```
Comment (bdach) [inline wiki/Difficulties/en.md]: This alt-text doesn't seem all that useful - how about something like
```suggestion
![An example of how a beatmap's star rating is displayed in the game.](img/difficulty-example.jpg)
```

File: wiki/Difficulties/en.md [hunk 3]
```diff
@@ -11,13 +11,14 @@ tags:
...
 
 ## Difficulty levels
 
 *See also: [Difficulty Naming](/wiki/Ranking_Criteria/Difficulty_Naming)*
 
-These are the typical levels of difficulty that a beatmap can fall under. Specific definitions and requirements of each can be found in their respective mode's [ranking criteria](/wiki/Ranking_Criteria).
+These are the most common beatmap difficulty names that can be found on almost every map. Specifics and definitions can be found under the specific mode's [ranking criteria](/wiki/Ranking_Criteria).
```
Comment (bdach) [inline wiki/Difficulties/en.md]: Repetition again:
> *Specifics* and definitions (...) under the *specific* mode's

The introductory sentence could be also changed to something like
> The most common beatmap difficulty names for existing gameplay modes are listed below.

File: wiki/Difficulties/en.md [hunk 4]
```diff
@@ -11,13 +11,14 @@ tags:
 
 *Not to be confused with [Beatmaps](/wiki/Beatmaps).*
 
-A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult it is.
+A beatmap's **difficulty** describes the level of skill needed to complete the beatmap. The difficulty name and star rating of a beatmap usually describe the level of skill needed to pass the beatmap.
```
Comment (cl8n) [inline wiki/Difficulties/en.md]: "it" unambiguously refers to the same subject, no need to repeat

File: wiki/Difficulties/en.md [hunk 5]
```diff
@@ -11,13 +11,14 @@ tags:
...
+![An example of how a beatmap's star rating is displayed in the game.](img/difficulty-example.jpg)
```
Comment (cl8n) [inline wiki/Difficulties/en.md]: an example showing multiple difficulties is probably better, and with a set that has diff names relating to their difficulty (easy normal hard etc)

File: wiki/Difficulties/en.md [hunk 6]
```diff
@@ -63,7 +64,7 @@ These are the typical levels of difficulty that a beatmap can fall under. Specif
 
 ![Star rating ranges](img/SR-range.png)
 
-On the website, beatmaps are automatically given a difficulty icon based on these star rating ranges:
+On the website, beatmaps are automatically given a difficulty icon based on star rating:
```
Comment (cl8n) [inline wiki/Difficulties/en.md]:
```suggestion
On the website, beatmaps are automatically given a difficulty icon based on their star rating:
```

**Pull request closed without merging**